ONBOARDING.rst

```diff
@@ -0,0 +1,29 @@
+=========================================
+Onboarding to LSST Strong lensing project
+=========================================
+
+This GitHub project originates from an inter-collaboration agreement between
+the two LSST Science collaborations:
+
+- `Strong Lensing Science Collaboration (SLSC) <https://sites.google.com/view/lsst-stronglensing?pli=1>`_
+- `Dark Energy Science Collaboration (DESC) <https://lsstdesc.org>`_
+
+You need to sign up for either one or both Science Collaboration to have access to LSST resources, communications and the Slack channel.
+We encourage you to sign up for both science collaborations.
+
+
+We have setup a document outlining how the LSST strong lensing simulation infrastructure could look like
+and what principles to follow to develop `here <https://docs.google.com/document/d/1dlwi7vP0jIL_miOF8VRiWd_0sALR4tklZlIIPuBJz9U/edit?usp=sharing>`_.
```

Comment:
Can we add the generic template that i extracted from your sims doc so it'ss here for others to use as their templatess for elements of the pipeline? https://docs.google.com/document/d/1aCyo0D9XF48VNT7l8OvGBgxP3V4SLRlPxAJSrm2-ubg/edit